### ISSUE TITLE
Leftover VM in jenkins

### ISSUE THREAD

**Issue by tgraf** [cilium/cilium#1070]
https://jenkins.cilium.io/job/cilium/job/cilium/job/PR-1064/4/execution/node/23/log/

```
[cilium_cilium_PR-1064-RVM3BNOILLZLMOTHCJSMSMZKKUP4LMWTARNRY2LNHVZ2WGS2SILA] Running shell script
+ ./tests/k8s/start

cd ./tests/k8s
+ cd ./tests/k8s
DOCKER_IMAGE_TAG=local_build RUN_TEST_SUITE=1 vagrant up
+ DOCKER_IMAGE_TAG=local_build
+ RUN_TEST_SUITE=1
+ vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Checking if box 'bento/ubuntu-16.10' is up to date...
==> default: A newer version of the box 'bento/ubuntu-16.10' is available! You currently
==> default: have version '2.3.5'. The latest is version '2.3.7'. Run
==> default: `vagrant box update` to update.
==> default: Clearing any previously set forwarded ports...
==> default: Fixed port collision for 22 => 2222. Now on port 2206.
==> default: Clearing any previously set network interfaces...
There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr is shown below.

Command: ["modifyvm", "da39106c-03b3-418f-81d6-31cfff582e32", "--nic2", "none", "--nic3", "none", "--nic4", "none", "--nic5", "none", "--nic6", "none", "--nic7", "none", "--nic8", "none"]

Stderr: VBoxManage: error: The machine 'k8s_default_1499079518809_48265' is already locked for a session (or being unlocked)
VBoxManage: error: Details: code VBOX_E_INVALID_OBJECT_STATE (0x80bb0007), component MachineWrap, interface IMachine, callee nsISupports
VBoxManage: error: Context: "LockMachine(a->session, LockType_Write)" at line 507 of file VBoxManageModifyVM.cpp
```

**Comment by aanm** [comment]
It's related with https://github.com/cilium/cilium/issues/1047

**Comment by ianvernon** [comment]
That is correct - #1045 fixes this issue. I am actively checking the state of the Jenkins slaves until it is merged to keep things running smoothly.

**Comment by ianvernon** [comment]
Closed as this was fixed as part of #1045 .
